client/src/App.tsx

```diff
@@ -169,6 +173,9 @@ const App = () => {
     sseUrl,
     env,
     bearerToken,
+    headerName,
+    proxyServerUrl: getMCPProxyAddress(config),
```

Comment:
these are no longer their in `main`, please check

Reply:
Thanks, I'll remove it now